### PR TITLE
Improve a bit clarity in README.md

### PR DESCRIPTION
**Pull request by Laurentiu-Andronache**
# Task:

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Document update
- [ ] Quality of life improvement

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have NATSPEC'd any new functions
- [ ] If appropriate, I have properly tested my new functionality

### CODE REVIEW

**Comment by kjr217** [comment]
Can you undo the linting you did on the README?

**Comment by Laurentiu-Andronache** [comment]
done, sorry, I had prettier plugin installed in VScode and it's set up to handle all files

File: README.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ At Rysk, we use Yarn for package management with Yarn workspaces. To get started
 npm install --global yarn
 ```
 
+We use jq for formatting JSON files from the command line. To install it, you might have to run:
+
+```sh
+$ sudo apt-get install jq
```
Comment (kjr217) [inline README.md:30]: I think this is only applicable for linux, for macos i needed to do brew install jq

Comment (Laurentiu-Andronache) [reply README.md:30]: well I did say "you might have to run"; each will install however he needs to install

**Comment by MeanBoyCousin** [comment]
@Laurentiu-Andronache - Don't forget us lowly DOS users! Had to use Chocolatey to install `jq` on Windows.

**Pull request merged**